devicetypes/axis/axis-gear-st.src/axis-gear-st.groovy

```diff
@@ -199,27 +200,33 @@ def off() {
 //Command to set the blind position (%) and log the event
 def setLevel(value, rate=null) {
 	log.info "setLevel ($value)"
-	
+
+	setShadeLevel(value)
+}
+
+def setShadeLevel(value) {
+	log.info "setShadeLevel ($value)"
 	Integer currentLevel = state.level
-	
+
 	def i = value as Integer
-	sendEvent(name:"level", value: value, displayed:true)
-	
+	sendEvent(name:"level", value: value, unit:"%", displayed:true)
```

Comment:
Should we change `displayed` to `false` here so it doesn't look like duplicate events in history?